styles.css

```diff
@@ -153,10 +169,125 @@ nav ul li a:hover {
 Cow visiting the Statue of Liberty
 */
 
+/* PORTFOLIO ITEMS STYLES */
+
+.portfolio {
+	background-color: #000000;
+	color: #ffffff;
+}
+
+.portfolio .container {
+	padding-top: 30px;
+}
+.portfolio h2 {
+	margin: 0 0 20px 0;
+	text-align: center;
+	font-size: 54px;
+}
+
+.portfolio h3 {
+	font-size: 25px;
+	margin-bottom: 5px;
+}
+
+.portfolio-text {
+	padding-top-top: 20px;
+}
+
+#portfolio-item-box {
+	text-align: center;
+}
+
+.portfolio-item {
+	display: inline-block;
+	margin: 20px auto 5px auto;
+	margin-right: 5px;
+	margin-left: 5px;
+}
+
+.portfolio-item div {
+	width: 365px;
+	height: 225px;
+	background-color: #fefefe;
+	border-radius: 10px;
+}
+
+.portfolio-item a {
+	font-size: 20px;
+}
+/*PORTFOLIO ITEM IMAGES*/
+
+#pf-item-1-img {
+	background-image: url(images/item1.png);
+	background-position: center;
+  	background-repeat: no-repeat;
+  	background-size: cover;
+  	position: relative;
+}
```

Comment:
image 1 shares a lot with 2 to 6. To reduce bloat you could make a 'portfolio' tag for all the background stuff, then 'item-1' just has its own background image